modules/deploy/pages/deployment-option/cloud/create-topic.adoc

```diff
@@ -1,6 +1,5 @@
 = Create a Topic
 :description: Learn how to create a topic for a Redpanda Cloud cluster.
-:page-aliases: cloud:create-topic.adoc
 :page-cloud: true
```

Comment:
Why did we remove the alias? `:page-aliases: cloud:create-topic.adoc`

Reply:
idk, I've forgotten why I did or didn't do things along the way. But it's back now!